app/code/Magento/InventorySourceSelectionApi/Model/Algorithms/Result/GetDefaultSortedSourcesResult.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySourceSelectionApi\Model\Algorithms\Result;
+
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\InventoryApi\Api\SourceItemRepositoryInterface;
+use Magento\InventorySourceSelectionApi\Api\Data\InventoryRequestInterface;
+use Magento\InventorySourceSelectionApi\Api\Data\SourceSelectionResultInterface;
+use Magento\InventorySourceSelectionApi\Api\Data\SourceSelectionItemInterfaceFactory;
+use Magento\InventorySourceSelectionApi\Api\Data\SourceSelectionResultInterfaceFactory;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+
+/**
+ * Return a default response for sorted source algorithms
+ */
+class GetDefaultSortedSourcesResult
```

Comment:
why do we have this class in the InventorySourceSelectionApi module ? 
looks like a wrong place to it

Reply:
Uhm... I think I just misplaced it while refactoring the code.

Reply:
no, looks like it's common implementation, so Api/Model is a correct place, just the name of this service is misleading